metadata-ingestion/docs/sources/dbt/dbt-cloud_recipe.yml

```
@@ -6,9 +6,9 @@ source:
     # In the URL https://cloud.getdbt.com/next/deploy/107298/projects/175705/jobs/148094,
     # 107298 is the account_id, 175705 is the project_id, and 148094 is the job_id
 
-    account_id: # set to your dbt cloud account id
-    project_id: # set to your dbt cloud project id
-    job_id: # set to your dbt cloud job id
+    account_id: '107298' # set to your dbt cloud account id
```

make it clear that this is not something they can just copy
```suggestion
    account_id: "${DBT_ACCOUNT_ID}" # set to your dbt cloud account id
```